pythonx/UltiSnips/snippet_manager.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, expand_trigger, forward_trigger, backward_trigger):
         enable_snipmate = True
         if _vim.eval("exists('g:UltiSnipsEnableSnipMate')") == "1":
             enable_snipmate = _vim.eval("g:UltiSnipsEnableSnipMate")
-        if enable_snipmate:
+        if enable_snipmate == "1":
```

Comment:
it was my first try, but it ends with error, when someone set "true"/"false"/'False"
and in unexpected behaviour when set to e.g. "3"   

IMHO it is also more coherent with docs, that explicitly says: only "1" value enable this feature

int(v) should be additionally backed by handling ValueError exception and that definitely is overhead to so simple functionality